Calendar/index.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import React, { useState, useCallback } from 'react';
+import { View, StyleSheet, Dimensions } from 'react-native';
+import Header from './Header';
+import Month from './Month';
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 10,
+  },
+});
+
+function getMonthCount(width) {
+  return width > 400 ? 2 : 1;
+}
+
+function getStartOfMonth(date) {
+  const first = new Date(date.getFullYear(), date.getMonth(), 1).getTime();
+  return first - new Date(first).getDay() * 86400 * 1000;
+}
+
+type Props = {
+  renderDay: React.Node,
```

Comment:
it should be `renderDate` for consistency and its a function (date: Date) => React.Node